docs/UserGuide.md

```diff
@@ -9,32 +9,57 @@ title: User Guide (v1.3)
- find or filter for applicants with ease and otherwise speeds up the application process.
+TAS have many functions that university administrative staff will find to be useful.
+
+These functions include:
+1. Pinning of important applicants. 
+2. Easy edit of applicant's detail.
+For example, you typed in the wrong name.You can easily edit the wrong name to the correct one.
+3. Finding a particular applicant. 
+For example, you can find a particular applicant in a large list of over 100 applicants just by keying in the applicant's name.
+4. Filtering of applicants by a certain trait, for instance application status. 
+Filtering by application status divides the list to 3 portions according to the 3 different types application status. 
+
+These functions will speed up the application process immensely.
 
 TAS is **optimised for use via a Command Line Interface (CLI)** while simultaneously having the benefits of a **Graphical User Interface
 (GUI)**. If you can type fast, TAS can handle your scholarship application process faster than conventional GUI apps.
 
 #### Who is this User Guide for?
 
-We have created this user manual for people who want to learn how to use our app.
-It is best suited for administrative staff members who seek an app to streamline their scholarship management procedure.
+It is stated that on average 1.7 million university scholarships are given out each year in the United States alone.Furthermore, 
+it is found that only 1 in 8 student actually wins a scholarship.This brings us to about more than 10 million applications a year.
+
+As such, many universities often take an extended period of many months to process these applications.
+University administrative staff often have to work overtime to process this large amount of applications.
+Poor scholarship management procedures employed in many of these universities further complicates the process. 
+
+TrackAScholar (TAS) is an app designed for university administrative staff members to streamline the scholarship management procedure. .
+
+TrackAScholar (TAS) seeks to 
+1. Reduce the time taken for university to process the scholarships
+2. Improve the current scholarship management process
+3. Decrease the number of times staff members work overtime 
 
-Alternatively, you may read our [Developer Guide](https://ay2223s1-cs2103t-w10-3.github.io/tp/DeveloperGuide.html)
- to find out more about the technical components of our app.
+For technical upkeep, please refer to our [Developer Guide](https://ay2223s1-cs2103t-w10-3.github.io/tp/DeveloperGuide.html)
+
+#### Current version
+This is the third iteration of the user guide.
+The current version is version 1.3.
 
 [Return to top](#table-of-contents)
 
 --------------------------------------------------------------------------------------------------------------------
 
-## Legend
+## Legend - explanation of signs which may help the user better understand the documentation
```

Comment:
Legend is also self-explanatory, although if you want to explain it further, put it in a new line, not in the header.

Reply:
Ok sure thank you.I have moved this below